MAUI/Picker/accessibility.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Accessibility with .NET MAUI Picker Control | Syncfusion
+description: Learn here all about accessibility with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
+control: SfPicker
+documentation: ug
+---
+
+# Accessibility in .NET MAUI Picker (SfPicker)
+
+The .NET MAUI Picker control has interact with the `Header`, `Column Header`, `Footer`, and `Picker Items`.
+
+## Header Layout
+
+The SfPicker control has intract header text with localization of header layout.
+
+## Column Header Layout
+
+The SfPicker control has intract column header text with localization of column header layout.
+
+## Footer Layout
+
+The SfPicker control has intract validation buttons (OK and Cancel) by setting the OkButtonText and CancelButtonText properties of the PickerFooterView with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".
```

Comment:
The SfPicker control has interact validation buttons (OK and Cancel) by setting the OkButtonText and CancelButtonText properties of the PickerFooterView with localization of footer layout. The Default value of the OkButtonText property is "OK", and CancelButtonText is "Cancel".

Reply:
Changed